mshadow/tensor_cpu-inl.h

```diff
@@ -309,7 +309,7 @@ inline void SoftmaxGrad(Tensor<cpu, 2, DType> dst,
 #pragma omp parallel for
   for (openmp_index_t y = 0; y < dst.size(0); ++y) {
```

Comment:
Does OMP no longer expect the limit var to be a scalar?  Maybe I am old, but it used to be a requirement.